CONTRIBUTING.md

```diff
@@ -2,44 +2,41 @@
 
 ##Reporting Issues
 
-See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and format to issue reports.
+To report an issue, read the format in the text box when you create one! Be descriptive, provide images if there is something visual, and if possible test the issue yourself. An issue must be replicatable or it will be closed!
```

Comment:
Could we add something like "where possible please list steps to reproduce issue"